locale/es/docs/guides/security/index.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Buenas prácticas de Seguridad en Node.js
+layout: docs.hbs
+---
+
+# Buenas prácticas de seguridad en Node.js
+
+## Propósito
+
+Este documento tiene la intención de extender el [modelo de amenazas][] actual y proporcionar directrices para asegurar una aplicación Node.js.
+
+## Contenido del documento
+* Mejores prácticas: Mostramos una forma simplificada y resumida de ver las mejores prácticas. Podemos usar [este issue][issue sobre la guía de seguridad] o [esta guía][nodejs guideline] como punto de partida. Es importante señalar que este documento es específico a Node.js, si está buscando algo amplio, considere [OSSF Best Practices][].
+* Ataques explicados: ilustrar y documentar con lenguaje sencillo con algunos ejemplos código
+  (cuando sea posible) los ataques que estamos mencionando en el modelo de amenazas.
```

Comment:
**suggestion:** `los ataques que estamos mencionando en el modelo de amenazas.` --> `los ataques mencionandos en el modelo de amenazas.`

Reply:
Fixed in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)